spring-core/src/main/java/com/baeldung/factorybean/FactoryBeanAppConfig.java

```diff
@@ -6,11 +6,23 @@
 @Configuration
 public class FactoryBeanAppConfig {
     
+<<<<<<< HEAD
+    @Bean
+=======
     @Bean(name = "tool")
+>>>>>>> f354f9e3057481c76c10a1e2aa127d3abb95b329
```

Comment:
@pivovarit  damn, let me work on them... this has been quite common on my pulls. Tell me how I can pull and update my local repo without adding merge conflicts :)